test/parallel/test-http-client-aborted-event.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+
+const server = http.Server(function(req, res) {
+  console.log('Server accepted request.');
```

Comment:
I've got no problem at all with seeing the `console.log()` calls removed. But I'm curious about the TAP comment. I'm pretty sure `test.py` or whatever captures the test's stdout and stderr and does not interleave it with TAP output. Is there a subtlety I'm missing where a `console.log()` can cause problems with TAP in that case? Many (perhaps even most) of our tests do `console.log()` and/or `console.error()`. So if this is A Thing, I'd definitely like to know more. (/cc @nodejs/testing)